src/DDTrace/Propagators/CurlHeadersMap.php

```diff
@@ -1,71 +0,0 @@
-<?php
-
-namespace DDTrace\Propagators;
-
-use DDTrace\NoopSpanContext;
-use DDTrace\Propagator;
-use DDTrace\Sampling\PrioritySampling;
-use DDTrace\Contracts\SpanContext;
-use DDTrace\Contracts\Tracer;
-
-/**
- * A propagator that inject distributed tracing context in curl like indexed headers arrays:
- * ['header1: value1', 'header2: value2']
- */
-final class CurlHeadersMap implements Propagator
-{
-    /**
-     * @var Tracer
-     */
-    private $tracer;
-
-    /**
-     * @param Tracer $tracer
-     */
-    public function __construct(Tracer $tracer)
-    {
-        $this->tracer = $tracer;
-    }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function inject(SpanContext $spanContext, &$carrier)
-    {
-        foreach ($carrier as $index => $value) {
-            if (
-                strpos($value, Propagator::DEFAULT_TRACE_ID_HEADER) === 0
-                || strpos($value, Propagator::DEFAULT_PARENT_ID_HEADER) === 0
-                || strpos($value, Propagator::DEFAULT_BAGGAGE_HEADER_PREFIX) === 0
```

Comment:
I don't think we have any tests for `DEFAULT_BAGGAGE_HEADER_PREFIX` outside of `CurlHeadersMapTest` which is being removed in this PR. I think OpenTracing headers was something that wasn't moved to the OpenTracing implementation a while back. But if we delete this, does this means we also drop support for propagating `ot-baggage-*` headers for curl calls?

Reply:
We already have a couple versions ago (but I didn't notice) Here I'm just dropping the old code which hasn't been used since a month (PHP 5, for other versions - longer).
I'd like to note here that our internal span does not have any concept of baggage...